convert2rhel/subscription.py

```diff
@@ -469,7 +431,10 @@ def __call__(self):
 
             try:
                 if self.password:
-                    loggerinst.info("Registering via username/password: %s" % " ".join(utils.hide_secrets(self.args)))
+                    loggerinst.info("Organization: %s", self.org)
+                    loggerinst.info("Activation Key: %s", "*" * 5)
```

Comment:
I think we want to change it here to use `username` and `password` instead of `activation key`, as it seems that the `activation key` one is only the one that happens in the `else` statement... 

```suggestion
                    loggerinst.info("Username: %s", self.username)
                    loggerinst.info("Password: %s", "*" * 5)
```

Reply:
Good catch, i'll change those lines